### PR TITLE
Add roadmap

### PR DESCRIPTION
**Pull request by trask**
This is a high-level list based on combing through the current issue list.

Appreciate any reviews/comments/ideas!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,3 +197,22 @@ To turn on the agent's internal debug logging:
...
+
+High-level roadmap as we work towards a 1.0 release:
+
+* Keep up with the specifications, including but not limited to:
+  * Implement all applicable semantic attributes
+    * Clearly document any additional attributes that are set
+        which are not defined by semantic conventions
+  * Standard metrics (still TBD, e.g.
+      [opentelemetry-specification#522](https://github.com/open-telemetry/opentelemetry-specification/issues/522))
+* Build out manual instrumentation for all existing auto-instrumentation
```
Comment (iNikem) [inline README.md:211]: Why this is needed for 1.0?

**Comment by iNikem** [comment]
Better/simple configuration for propagators/exporters/samplers?

**Comment by jeffalder** [comment]
Should there be an associated GH project / milestone so that the appropriate issues to get to 1.0 can be documented and tracked?

File: README.md (hunk 1 above)
Comment (trask) [reply README.md]: Added:

> It is our goal to release 1.0 (GA) of the auto-instrumentation agent during the first wave of
OpenTelemetry 1.0 (GA) releases, along with as many manual instrumentation libraries as possible.

and removed the word `all` from this specific bullet point.

**Comment by trask** [comment]
> Should there be an associated GH project / milestone so that the appropriate issues to get to 1.0 can be documented and tracked?

That's a good question. I added to Thu [SIG meeting agenda](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/edit) to discuss.

We're currently using milestones just to plan one month at a time: https://github.com/open-telemetry/opentelemetry-java-instrumentation/milestone/4

I did go through and label a bunch of issues, I think everything with these labels can be considered targeted for 1.0:

* [specification](https://github.com/open-telemetry/opentelemetry-java-instrumentation/labels/specification)
* [documentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/labels/documentation)
* [packaging](https://github.com/open-telemetry/opentelemetry-java-instrumentation/labels/packaging)
* [bug](https://github.com/open-telemetry/opentelemetry-java-instrumentation/labels/bug)
* [sporadic test failure](https://github.com/open-telemetry/opentelemetry-java-instrumentation/labels/sporadic%20test%20failure)

**Comment by iNikem** [comment]
I think having GA/1.0 milestone with associated tasks is a very good idea.

**Comment by trask** [comment]
> I think having GA/1.0 milestone with associated tasks is a very good idea.

Trying to think how this would work:

Setting the milestone for all of these issues to 1.0, and then moving the issues to an earlier milestones as needed, e.g. when planning 0.5 release?

**Comment by iNikem** [comment]
I don’t see a need in planning earlier releases and even assigning fixed issues to releases before 1.0. I would create 1.0 milestone, assign all tickets that we want to be fixed before GA to that milestone and that’s it. Most of the tickets will be fixed earlier and released on various point releases. So what? :)

**Pull request merged**